.circleci/config.yml

```diff
@@ -80,9 +101,8 @@ commands:
                   - /usr/local/Homebrew
                   - ~/Library/Caches/Homebrew
 
-      - node/install-packages:
```

Comment:
What was wrong with this one? :eyes: 

Was it not caching cypress correctly?

Reply:
Yes, I linked it in the description, multiple cache locations are not supported and cannot be supported because of circle limitations